docs/build/wasm/from-zero-to-ink-hero/nft/Wizard/wizard.md

```diff
@@ -76,25 +79,17 @@ Your `Cargo.toml` should now look like this:
 ```toml
 [package]
 name = "shiden34"
-version = "1.0.0"
+version = "3.0.0"
+authors = ["Astar builder"]
 edition = "2021"
-authors = ["The best developer ever"]
 
 [dependencies]
-
-ink_primitives = { version = "~3.4.0", default-features = false }
-ink_metadata = { version = "~3.4.0", default-features = false, features = ["derive"], optional = true }
-ink_env = { version = "~3.4.0", default-features = false }
-ink_storage = { version = "~3.4.0", default-features = false }
-ink_lang = { version = "~3.4.0", default-features = false }
-ink_prelude = { version = "~3.4.0", default-features = false }
-ink_engine = { version = "~3.4.0", default-features = false, optional = true }
+ink = { version = "~4.0.0", default-features = false}
 
 scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
-scale-info = { version = "2", default-features = false, features = ["derive"], optional = true }
+scale-info = { version = "2.3", default-features = false, features = ["derive"], optional = true }
 
-# Include brush as a dependency and enable default implementation for PSP34 via brush feature
-openbrush = { tag = "v2.3.0", git = "https://github.com/Supercolony-net/openbrush-contracts", default-features = false, features = ["psp34", "ownable"] }
+openbrush = { tag = "3.0.0", git = "https://github.com/727-Ventures/openbrush-contracts", default-features = false, features = ["psp34", "ownable"] }
 
 [lib]
 name = "shiden34"
```

Comment:
same, https://use.ink/faq/migrating-from-ink-3-to-4/#removal-of-libcrate-type-and-libname-from-contract-manifest

Reply:
ok